src/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+const getBaseUrl = (): string => {
+    const url = window.location.href.split('plugins')[0];
+    return url.substring(0, url.length - 1);
+};
```

Comment:
```suggestion
const getBaseUrls = (): {mattermostServer: string, plugin: string} => {
    const url = window.location.href.split('plugins')[0];
    const mattermostServer = url.substring(0, url.length - 1);
    const plugin = `${mattermostServer}/plugins/${PLUGIN_ID}/api/v1/teams`
    
    return {mattermostServer, plugin}
};
```

this way we can use this single util to get both types of URL wherever needed